src/containers/common/ClassroomManagerContainer.jsx

```diff
@@ -35,6 +37,10 @@ export class ClassroomManagerContainer extends React.Component {
   resetToastState() {
     this.setState({ toast: { message: null, status: null } });
   }
+  
+  selectClassroom(classroom) {
```

Comment:
To deselect the active Classroom, call `selectClassroom(null)`. As mentioned in the PR comments, let me know if it makes more sense to simply have a`deselectClassroom()` that does the same thing but may be easier to read for our fellow hoo-manns.